jobs/involuntary-dissolutions/devops/vaults.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "vault": "shared",
+        "application": [
+            "nats"
+        ]
+    },
+    {
+        "vault": "entity",
+        "application": [
+            "postgres-legal",
+            "sentry",
+            "launchdarkly"
+        ]
+    }
```

Comment:
Think we'll need to add `nats-filer`.

We'll need to probably publish a message to the filer topic so involuntary dissolution filings are processed.